online-bookstore/online-bookstore/src/main/java/com/example/online_bookstore/repository/UserAccountRepo.java

```diff
@@ -12,12 +12,13 @@
 @Repository
 public interface UserAccountRepo extends JpaRepository<UserAccount,Long> {
 
-     List<UserAccount> findByLastnameOrFirstname(String lastname , String firstname);
+     List<UserAccount> findByLastnameOrFirstname(String lastname, String firstname);
 
      Optional<UserAccount>findByUsername(String username);
 
      List<UserAccount> findAllByOrderByUserIdAsc();
 
+
     @Query("select u from UserAccount u where u.role != :ADMIN " +
             "order by u.Id asc ")
     List<UserAccount> findAllNonAdmin(Role admin);
```

Comment:
format this query by separating SQL STATEMENTS in separate lines eg.
select ...
from ...
where ... and ...